database/database.js

```diff
@@ -0,0 +1,22 @@
+const connection = process.env.DATABASE_URL || 'psql://localhost:5432/ribbit';
+
+const repos = {
+  rooms: require('./repos/rooms.js'),
+  users: require('./repos/users.js'),
+  chained: require('./repos/chained.js')
+};
+
+const options = {
+  extend: (obj) => {
+    const keys = Object.keys(repos);
+    keys.forEach((key) => {
+      obj[key] = repos[key](obj, pgp);
```

Comment:
Would also like some comments here in the functional programming with slightly vague names, which, while quite clever, is hard to grok.